wolox-books/src/views/Register.vue

```diff
@@ -0,0 +1,102 @@
+<template lang='pug'>
+  .container
+      .container-register
+        img.logo(alt='Wolox logo' src='../assets/logo.png')
+        form(@submit.prevent='onSubmit').container-form
+          .container-field(v-for='(field, name) in fields' :key='name')
+            label.label(:for='name')
+              | {{field.label}}
+            input.input(:type='field.type' v-model='field.value' :id='name')
+          button.button-sign-up(:type='button') Sign Up
```

Comment:
No need for the bind in here, just `type='button') is enough. And remember to write the plain text in pipe notation:

```pug
button.button-sign-up(:type='button')
  |  Sign Up
```